src/libs/OptionsListUtils.ts

```diff
@@ -509,6 +518,28 @@ function getLastActorDisplayName(lastActorDetails: Partial<PersonalDetails> | nu
         : '';
 }
 
+/**
+ * Update alternate text option when applicable
+ */
+function getAlternateTextOption(
```

Comment:
What's the alternative text option?

Reply:
right, wrong naming here. Updated it with `getAlternateText`